<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LTIN122556" ignored="0" name="password_Rest-Run1" tests="1" failures="0" timestamp="31 Dec 2019 15:30:29 GMT" time="73.589" errors="0">
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="com.mop.qa.testbase.TestBase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.mop.qa.testbase.TestBase.afterMethod(TestBase.java:232)
	at com.mop.qa.testrunner.TestRunner.main(TestRunner.java:167)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="Test_NFT_S3_Password_Reset_TestCase" time="68.944" classname="com.mop.qa.test.regression.Test_NFT_S3_Password_Reset"/>
</testsuite> <!-- password_Rest-Run1 -->
